fireant/widgets/highcharts.py

```diff
@@ -89,29 +96,25 @@ def transform(
         :return:
             A dict meant to be dumped as JSON.
         """
-        colors = itertools.cycle(self.colors)
-
-        is_timeseries = dimensions and dimensions[0].data_type == DataType.date
+        result_df = data_frame.copy()
 
         dimension_map = {
-            dimension_alias: dimension
-            for dimension_alias, dimension in zip(data_frame.index.names, dimensions)
+            alias_selector(dimension.alias): dimension for dimension in dimensions
         }
-        num_dimensions = len(dimension_map)
-        dimension_fields = [
-            dimension_map[field_alias]
-            for field_alias in data_frame.index.names[:num_dimensions]
-        ]
+
+        colors = itertools.cycle(self.colors)
+
+        is_timeseries = dimensions and dimensions[0].data_type == DataType.date
 
         # Timestamp.max is used as a marker for rolled up dimensions (totals). Filter out the totals value for the
```

Comment:
I am not sure why we actually filter this out. If a user specifies totals, it'd make sense to show it on the chart IMO. Anyway, not one for this PR.